compiler-plugin/src/main/java/io/ballerina/stdlib/websubhub/task/ListenerInitAnalysisTask.java

```diff
@@ -55,6 +55,9 @@ public void perform(SyntaxNodeAnalysisContext context) {
         SyntaxKind nodeSyntaxKind = node.kind();
         if (nodeSyntaxKind == SyntaxKind.EXPLICIT_NEW_EXPRESSION) {
             ExplicitNewExpressionNode expressionNode = (ExplicitNewExpressionNode) node;
+            if (!(expressionNode.typeDescriptor() instanceof QualifiedNameReferenceNode)) {
```

Comment:
Instead of an `instanceof` check, shall we use the following check?

The `instanceof` is a comparably expensive operation, and the `TypeKind`, `SyntaxKind`, etc. are there to avoid these checks.

```suggestion
            if (!(expressionNode.kind() == SyntaxKind.QUALIFIED_NAME_REFERENCE)) {
```